misc/python/materialize/feature_benchmark/benchmark_versioning.py

```diff
@@ -30,7 +30,7 @@
 ] = "d1e72837a342c3ebf1f4a32ec583b1b78a78644cdba495030a6df45ebbffe703"
 SHA256_BY_SCENARIO_FILE[
     "optbench.py"
-] = "e087c72dbda9c48d9de14917b3c40a1f8b07617a3b406d52db917318c8057af3"
+] = "c7b8b97cb77ed8cebb74b9236da707cc516f928526a1ef440b89e367f5fc6898"
```

Comment:
Sorry 🙈 
We will discuss this next Tuesday and either get rid of it or at least reduce the number of hashes.